### ISSUE TITLE
Question sabout fine-tuning a DeepAR model

### ISSUE THREAD

**Issue by jambudipa** [jdb78/pytorch-forecasting#1331]
I am confused by the documentation when trying to work out how to fine-tune a DeepAR model.

- How do I save a checkpoint once `trainer.fit()` has been called? Or, if they are saved automatically, how do I keep track of which is the most up-to-date?
- How do I prepare new data for use in fine-tuning?
  - do I need to be concerned with normalising the data when using `TimeSeriesDataSet()`? Will the normalisation from the previous checkpoint be considered?
  - does the `time_idx` column need to be in sequence with the previously trained data, or can it start from `0` with the new training set?

**Comment by jambudipa** [comment]
Ok,  thanks.So the sequence can pick up where it left off?And the normalisation must correspond to that of the domain and range of the original training data? Been a while since I’ve used a net like this.Sent from my iPhoneOn 18 Jun 2023, at 04:57, Chess_champion ***@***.***> wrote:﻿
Hi, The model saves the checkpoints by default, usually in the lightning_logs folder but you can change this.
You can normalize it through scalers or target_normalizer yourself.
The time_idex, I believe can be the new sequence of data with new train data.
